src/components/data/data-utils.js

```diff
@@ -152,8 +152,10 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
   const [additionalDeletions, setAdditionalDeletions] = useState([])
   const [deleting, setDeleting] = useState(false)
 
+  const selectedKeys = _.keys(selectedEntities)
+
   const doDelete = async () => {
-    const entitiesToDelete = _.concat(_.map(entityName => ({ entityName, entityType: selectedDataType }), selectedEntities), additionalDeletions)
+    const entitiesToDelete = _.concat(_.map(entity => ({ entityName: entity['name'], entityType: entity['entityType'] }), selectedEntities), additionalDeletions)
```

Comment:
For a literal string key, just do `entity.name`. The brackets are for when the key needs to be interpreted in some way.